modelforge/dataset/qm9.py

```diff
@@ -0,0 +1,204 @@
+import os
+from typing import Any, Dict, List, Tuple
+
+import gdown
+import h5py
+import numpy as np
+import torch
+from loguru import logger
+from .dataset import BaseDataset
+
+
+class PlainQM9Dataset(torch.utils.data.Dataset):
+    """
```

Comment:
What is the purposed of this class compared to !m9Dataset?

Reply:
I will remove this class.